lib/pescarte_web/templates/landing_html.ex

```diff
@@ -2,4 +2,19 @@ defmodule PescarteWeb.LandingHTML do
   use PescarteWeb, :html
 
   embed_templates("landing_html/*")
+
+  def handle_notice_text_length(text, max_length) do
+    case String.length(text) do
+      text_length when text_length >= max_length ->
```

Comment:
esse teste lógico está errado. o texto pode ter até o tamanho desejado, logo, se `max_length` for 110, significa que o tamanho máximo do texto é 110. se fizer `>=` significa que se o texto tiver 110 de tamanho, ele vai tbm vai ser truncado, o que não deveria ser verdade

```suggestion
      text_length when text_length > max_length ->
```